Makefile

```diff
@@ -153,16 +156,17 @@ image_builder/manifest.json: image_builder/gce.json image_builder/setup.sh lib.s
 	$(eval override _GAC_FILEPATH := $(call err_if_empty,GAC_FILEPATH))
 	$(call packer_build,$<)
 
-# TODO: Make this work.  Currently when qemu-kvm starts up, it complains
-# tries and fails to read files from my home directory on the host (outside
-# the container).
+# TODO: Document: Unless we assume $PWD is under the uses home-dir, accessing
+# sometimes necessary files like $HOME/.gitconfig or $HOME/.ssh won't work if
+# the container were to only volume-mount $PWD.
```

Comment:
(probably for later): I find it hard to parse this comment. If cwd is under HOME, how could you access `$HOME/.anything` by mounting `$PWD`? Would it make sense to comment instead something like `$HOME is needed for .config, .ssh, etc`?

Reply:
Long-story short: this command _use_ to volume mount `$PWD` only, took me hours to debug related problems.  It's likely  some of that debugging-think crept into my comment.  Yes, something along the lines of your suggestion would be better, but let's do it later so the images don't need to be re-built :grin:

Reply:
k. got this.